openpathsampling/pathmover.py

```diff
@@ -1624,33 +1855,36 @@ class MinusMover(WrappedMover):
     """
     _is_canonical = True
 
-    def __init__(self, minus_ensemble, innermost_ensembles, ensembles=None):
+    def __init__(self, minus_ensemble, innermost_ensemble):
         segment = minus_ensemble._segment_ensemble
-        try:
-            innermost_ensembles = list(innermost_ensembles)
-        except TypeError:
-            innermost_ensembles = [innermost_ensembles]
-        innermost_ensemble = paths.join_ensembles(innermost_ensembles)
-        subtrajectory_selector = RandomChoiceMover([
-            FirstSubtrajectorySelectMover(subensemble=segment,
-                                          n_l=minus_ensemble.n_l,
-                                          ensembles=[minus_ensemble]
-                                         ),
-            FinalSubtrajectorySelectMover(subensemble=segment,
-                                          n_l=minus_ensemble.n_l,
-                                          ensembles=[minus_ensemble]
-                                         ),
+        sub_trajectory_selector = RandomChoiceMover([
+            FirstSubtrajectorySelectMover(
+                ensemble=minus_ensemble,
+                sub_ensemble=segment,
+                n_l=minus_ensemble.n_l
+            ),
+            FinalSubtrajectorySelectMover(
+                ensemble=minus_ensemble,
+                sub_ensemble=segment,
+                n_l=minus_ensemble.n_l
+                ),
         ])
-        subtrajectory_selector.name = "MinusSubtrajectoryChooser"
+        sub_trajectory_selector.name = "MinusSubtrajectoryChooser"
 
-        repexs = [ReplicaExchangeMover(ensembles=[[segment, inner]])
-                  for inner in innermost_ensembles]
-        repex_chooser = RandomChoiceMover(repexs)
-        repex_chooser.name = "InterfaceSetChooser"
+        repex = ReplicaExchangeMover(
+            ensemble1=segment,
+            ensemble2=innermost_ensemble
+        )
```

Comment:
To clarify: you should change everything in this section back to plurals, and leave the list of repexs. Just change the `ensemble1=, ensemble2=` part. Everything else is a part of the changes I made so that this implements the multiple interface set minus move -- it was that way for a reason.